src/handlers/psp.ts

```diff
@@ -0,0 +1,436 @@
+import * as abi from '@api3/airnode-abi';
+import * as node from '@api3/airnode-node';
+import { ethers } from 'ethers';
+import { Dictionary } from 'lodash';
+import flatMap from 'lodash/flatMap';
```

Comment:
btw, you can do this here:
```
import {flatMap, isEmpty, isNil } from 'lodash';
```

Reply:
We've generally tried specifically NOT to do this as lodash doesn't have proper tree shaking (so you get the entire library)

Reply:
Noted, sorry, will avoid :)